handlebars.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  data: "./src/data",
+  layout: "./src/layout",
```

Comment:
Здесь необходимо указать
```suggestion
  layouts: "./src/layout",
```